### ISSUE TITLE
shells/ast-ksh fails on package creation

### ISSUE THREAD

**Issue by AttilaFueloep** [TritonDataCenter/pkgsrc-legacy#396]
Hi Guys,

another one liner which adds ksh-20120801nb1.
Should I generate the PR against upstream or joyent/pkgsrc?

**Comment by AttilaFueloep** [comment]
Created PR #9 against upstream, hope that's OK.

**Comment by mamash** [comment]
I cannot reproduce this problem and we've had a binary package for ast-ksh in 2016Q1 and 2016Q2 for SmartOS (ever since Richard's patch in March). Which Git branch are you building from?

(Real upstream is CVS, Joerg's Git mirror won't handle any PRs. Technically, neither won't we, but we can transfer patches over to CVS.)

**Comment by mamash** [comment]
For the record, [link to the PR](https://github.com/jsonn/pkgsrc/pull/9).

**Comment by AttilaFueloep** [comment]
Yes, now I realize that there is a binary package. I've no idea why failed to find it a month ago, maybe just a typo. Sorry for that. 

The build problem on the other hand still persists for me. Well, maybe I'm doing something wrong.
I'm buiding in a vm generated from the pkgbuild 16.2.0 image. Both, joyent/release/2016Q2 and current trunk are affected.

```
$ run-sandbox 2016Q2-x86_64
-> cd /data/pkgsrc-2016Q2/shells/ast-ksh/
-> bmake package
.....
ERROR: /opt/local/gcc49/lib/amd64/libssp.so.0: gcc49-4.9.3nb1 is not a runtime dependency
*** Error code 1
Stop.
bmake: stopped in /data/pkgsrc-2016Q2/shells/ast-ksh
*** Error code 1
```

Since there is a binary package and you can't reproduce it,  this looks like a misconfiguration on my side.
Feel free to close this PR, although i would be happy about any hints how to fix my build environment.

> Joerg's Git mirror won't handle any PRs.

Thanks, good to know. I'll close the PR then.

> Technically, neither won't we, but we can transfer patches over to CVS.

So is it OK to commit a PR here or would you prefer me to commit upstream?
